library/src/main/java/com/chuckerteam/chucker/internal/data/har/Har.kt

```diff
@@ -0,0 +1,42 @@
+package com.chuckerteam.chucker.internal.data.har
+
+import androidx.annotation.VisibleForTesting
+import com.chuckerteam.chucker.BuildConfig
+import com.chuckerteam.chucker.internal.data.entity.HttpTransaction
+import com.chuckerteam.chucker.internal.support.JsonConverter
+import com.google.gson.annotations.SerializedName
+import kotlinx.coroutines.Dispatchers
+import kotlinx.coroutines.withContext
+import java.text.SimpleDateFormat
+import java.util.Locale
+
+// http://www.softwareishard.com/blog/har-12-spec/
+// https://github.com/ahmadnassri/har-spec/blob/master/versions/1.2.md
+internal data class Har(
+    @SerializedName("log") val log: Log
+) {
+    object DateFormat : ThreadLocal<SimpleDateFormat>() {
```

Comment:
Because `SimpleDateFormat` is not thread safe, and these methods get called from coroutines which could potentially be executed from multiple threads. 

Do you have another way you'd like me to handle this?

Reply:
> Do you have another way you'd like me to handle this?

I think we should consider using `java.time` plus the core library desugaring: https://developer.android.com/studio/write/java8-support#library-desugaring

That would allow to use a `DateTimeFormatter` that is actually thread safe.

Reply:
Can do. I'm not familiar with it, but I'll look into it and get it incorporated.

Reply:
I would be wary of enabling desugaring. It will require consumers to enable desugaring in their apps as well. Also, it uses multidex and since Chucker `3.x` supports APIs below 21 it might be not the best experience for users. It seems rather invasive to support only this particular use case. To me, it seems like something that Chucker `4.x` could target.

Reply:
Yeah, I agree. I did this as a separate commit (5042a28da139458bf5b9c15d295dfa2cbde017a5), so it's easy to revert if we want.

Reply:
> It will require consumers to enable desugaring in their apps as well. Also, it uses multidex and since Chucker 3.x supports APIs below 21 it might be not the best experience for users

Good point 👍 I'm fine also staying with `SimpleDateFormat`. However, I think we have valid uses cases to use `java.time` in the future and push for desugaring. As @MiSikora that's definitely a thing to keep in mind for 4.x